lib/ace/ext/papeeria/biblio_completer.coffee

```diff
@@ -0,0 +1,117 @@
+foo = null # ACE builder wants some meaningful JS code here to use ace.define instead of just define
+define((require, exports, module) ->
+  LatexParsingContext = require("ace/ext/papeeria/latex_parsing_context")
+  Autocomplete = require('ace/autocomplete')
+  util = require("ace/autocomplete/util")
+  lang = require("../../lib/lang")
+
+  class BiblioFilesCompleter extends Autocomplete.Autocomplete
+    constructor: ->
+      super()
+    setFiles: (files) => @files = files
+    _setCompletions: =>
+      matches = @files.map((file) -> return {
+        name: file.name
+        file: file
+        meta: file.path
+        completer: this
```

Comment:
is it true that this is used only in the `Autocomplete#insertMatch` method? which btw is overridden in this class so it seems unnecessary to me